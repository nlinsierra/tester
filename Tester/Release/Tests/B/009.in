AS 3C 6H KS JS 5H 9C 8H AH 4H QS QD 9H TC 5C 6D TD QH 7S
