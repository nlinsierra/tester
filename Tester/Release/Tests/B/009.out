33
2C 4C 6C 7C 8C JC QC KC AC 2D 3D 4D 5D 7D 8D 9D JD KD AD 2H 3H 7H TH JH KH 2S 3S 4S 5S 6S 8S 9S TS
